templates/travis/.travis/script.sh.j2

```diff
@@ -13,11 +13,12 @@ export FUNC_TEST_SCRIPT=$TRAVIS_BUILD_DIR/.travis/func_test_script.sh
 # Gets set in .travis/settings.yml, but doesn't seem to inherited by
 # this script.
 export DJANGO_SETTINGS_MODULE=pulpcore.app.settings
+export PULP_SETTINGS=$TRAVIS_BUILD_DIR/settings.py
 
 if [ "$TEST" = 'docs' ]; then
-  {% if plugin_name == 'pulpcore' %}
+  {%- if plugin_name == 'pulpcore' %}
     export PULP_CONTENT_ORIGIN=http://$(hostname):24816
```

Comment:
Do we still need this with the fix above?

Reply:
seems to be found here (If `hostname` is the same):
https://travis-ci.com/github/pulp/pulpcore/jobs/316888406#L2846

Reply:
Are you asking if we need to set PULP_CONTENT_ORIGIN at all? Or if we should limit this to when testing pulpcore?

Which fix are you referring to? A fix that eliminates the need for PULP_CONTENT_ORIGIN? I am not sure what command later in script.sh uses it, the containers are already launched with it set as a config file setting.

Reply:
Well for one, why do we need it only for pulpcore? And second with using the `PULP_SETTINGS` variable above (Line 16), we make (among others) the `CONTENT_ORIGIN` setting available to dynaconf. If that is enough and set to the right value already, we could completely skip it here.